### ISSUE TITLE
Account not signed in mediated trade due to bug

### ISSUE THREAD

**Issue by partywave** [bisq-network/bisq#4946]
### Description

A trade between myself and a btc seller with a signed account was mediated due to a trade bug not permitting the seller to confirm receipt of funds. The trade was mediated successfully but as a result my buyer account was not signed.

The issue was discussed in support channels and referred here for possible bug report.

#### Version

v1.5.1

### Steps to reproduce

Test account signing after a mediated proceeding of trade state.

### Expected behaviour

Buyer account should be signed.

### Actual behaviour

Buyer account not signed.

#### Device or machine

MacOs 10.13

#### Additional info

Information available on request.

**Comment by MwithM** [comment]
If mediation was not produced due to a bug but another reason (i.e. wrong account) it would be ok to not sign the account.

**Comment by partywave** [comment]
Yes to clarify since this was caused by an apparent bug, the account should have been signed after mediation. Is there a way to retrospectively have my account signed due to successful trade?
